src/pug/blog/report_21.pug

```diff
@@ -0,0 +1,47 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #21
+
+block content
+  h2 Progress report #21
+  +news_header('8 March 2023', 'patents', '')
+
+  h3 Wireless Alpakka
+  p For the last months we have been researching (and developing) the wireless functionality on the Alpakka controller, originally we decided to continue using the #[em Raspberry Pi Pico], but moving to the #[em Pico-W] variant. Our initial work was to validate a potential BT solution, focusing on 4 key requirements:
+
+  h4 Latency
+  p Pico-W to Pico-W latency tests demostrated that, after some tuning, BT can achieve latencies down to #[em 4 milliseconds]. So in this regard BT had an acceptable performance for gaming.
+
+  h4 Interoperability
+  p One of the selling points of BT is that it works everywhere, unfortunately our tests have shown that even if you have a device that is following the BT specs, the results may be very different depending on which dongle is being used, which operating system, which drivers, etc. A workaround for this would be using a #[em second Pico-W as a dongle], which would be inconvenient but acceptable, as most gamer PC peripherals use custom dongles.
+
+  h4 Battery live
+  p We only conducted a single test with a wireless Alpakka prototype, in which we played for #[em 60 minutes], and a #[em 1200 mAh] battery went from #[em 4.18V to 4.12V], this was not representative of how long the battery would last for real, as battery voltage drop is not linear, but gives us confidence that 1200 mAh should be enough for a few gaming sessions, even in the current state of the firmware without power-saving optimizations.
+
+  h4 Consistency
+  p Unfortunately this is were the experiments with BT on the Pico-W failed to meet our requirements, after hundreds of hours trying, many tweaks, and even with the help of BlueKitchen devs, we were unable to get rid of #[em latency spikes] that often went into 20+ milliseconds, with big spikes up to 60+ milliseconds, that completely #[em ruined the gaming experience] with regular lagging.
+
+  img.mt.mr(src='/static/blog/wireless_setup.jpg' width='400px')
```

Comment:
Mabe move the pictures closer to where they are described. I.e., the first one close to the pico-w - pico-w above, and the 2nd one here with the power consumption / battery life